packages/@coorpacademy-components/src/test/helpers/render-component.js

```diff
@@ -34,7 +35,7 @@ const renderComponent = (t, Component, fixture) => {
 const renderNativeComponent = (t, Component, fixture) => {
   const {toJSON} = render(<Component {...fixture.props} />);
   const json = toJSON();
-  t.is(json.type, 'react-native-mock');
+  t.is(get('type', json), 'react-native-mock');
```

Comment:
to have a more specific error (more on ava level than @ script level).
From:
```
  Error thrown in test:
  TypeError {
    message: 'Cannot read properties of null (reading \'type\')',
  }
```
To:
```
  - undefined
  + 'react-native-mock'
```